tests/test_misc_table.py

```diff
@@ -606,6 +606,73 @@ def test_dtype_multiindex_single_level(
     assert db_new['misc'].index.dtype == expected_pandas_dtype
 
 
+def test_drop_and_pick_index():
+
+    table_id = 'misc'
+    index = pytest.DB[table_id].index[:5]
+    df_pick = pytest.DB[table_id].pick_index(index).get()
+    index = pytest.DB[table_id].index[5:]
+    df_drop = pytest.DB[table_id].drop_index(index).get()
+    pd.testing.assert_frame_equal(df_pick, df_drop)
```

Comment:
This would also be true, if both methods would return always an empty array.
Maybe add another check for the length of the dataframe?

Reply:
Good one. I had copied the test from `test_table.py`, but in fact the misc table has only 3 items so in both cases it returned all items. I adjusted the test and also the one in `test_table.py`.